src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTypeCheck.java

```diff
@@ -809,7 +809,7 @@ private boolean isCheckedMethod(DetailAST ast) {
         final String methodName =
             ast.findFirstToken(TokenTypes.IDENT).getText();
         return isVerifiable(ast) && !ignoredMethodNames.contains(methodName)
-                && !AnnotationUtil.containsAnnotation(ast, "Override");
+                && !AnnotationUtil.hasOverrideAnnotation(ast);
```

Comment:
This may be a breaking change, as this check has not previously supported FQDN annotations.

Reply:
@strkkk Please provide regression.

Reply:
Report is clean - https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/e081ae1_2022180404/reports/diff/index.html

Reply:
You changed many checks but only ran regression on one. You need to run regression on all the impacted checks.

Reply:
I can not believe that somebody user FQDN for good reason. It might, but in this case better not classify this update as minor.
@strkkk , please use "Pull #XXXX: " prefix

Reply:
I ran it on one check because it is the only one that has some change, for others there is no changes, only method extraction to util.

I added Pull prefix

Reply:
@rnveach see https://github.com/checkstyle/checkstyle/pull/11333#issuecomment-1049575520